test/Plutip.purs

```diff
@@ -55,21 +58,22 @@ config =
   -- Server configs are used to deploy the corresponding services:
   , ogmiosConfig:
       { port: UInt.fromInt 1337
-      , host: "localhost"
+      , host: "127.0.0.1"
       , secure: false
       }
   , ogmiosDatumCacheConfig:
       { port: UInt.fromInt 9999
-      , host: "localhost"
+      , host: "127.0.0.1"
       , secure: false
       }
   , ctlServerConfig:
       { port: UInt.fromInt 8081
-      , host: "localhost"
+      , host: "127.0.0.1"
       , secure: false
       }
-  , ogmiosDatumCachePostgresConfig:
-      { port: UInt.fromInt 5432
+  , postgresConfig:
+      { host: "127.0.0.1"
+      , port: UInt.fromInt 5432
```

Comment:
Please change the port to something else. It conflicts with ctl-runtime by default and I get a password prompt.

Reply:
did that